os/pm/pm.h

```diff
@@ -140,20 +140,14 @@ struct pm_global_s {
 
 	/* state       - The current state for this PM domain (as determined by an
 	 *               explicit call to pm_changestate())
-	 * recommended - The recommended state based on the PM algorithm.
 	 */
 
 	uint8_t state;
-	uint8_t recommended;
```

Comment:
won't be used this in lcd structure?
Please seperate pr about this.

Reply:
Its not needed for LCD structure